src/components/MainForm/Mainform.tsx

```diff
@@ -13,23 +13,29 @@ import DialogTitle from "@mui/material/DialogTitle";
 import InputLabel from "@mui/material/InputLabel";
 import FormControl from "@mui/material/FormControl";
 import Select, { SelectChangeEvent } from "@mui/material/Select";
-
 //React Hook Form
 import { useForm, SubmitHandler } from "react-hook-form";
-// const countries = [
-//   {
-//     value: "USD",
-//     label: "$",
-//   },
-//   {
-//     value: "AUS",
-//     label: "A$",
-//   },
-//   {
-//     value: "PAK",
-//     label: "PKR",
-//   },
-// ];
+
+//Some IntialStates
+let emailValid = false;
+let passValid = true;
+let submitBtn = true;
+
+const countries: any = {
```

Comment:
I see you've created countries array here and also on other component!
why?

Reply:
This array stores country flag names and their country code 
through this array i am getting the details